public/lib/record_mapper.rb

```diff
@@ -14,6 +14,20 @@ def repo_settings
         AppConfig[:aeon_fulfillment][self.repo_code]
     end
 
+    # If #show_action? returns false, then the button is shown disabled
+    # This method tests whether the button should be hidden (rather than just disabled)
+    # This is controlled by two repo config options, which default to false to maintain
+    # the existing functionality.
```

Comment:
The first line of this comment seems irrelevant to the `hide_button?` method, and could be moved to above the `show_action?` method. We believe that the following wording would be more appropriate for this method's documentation.

```
This method tests whether the button should be hidden. This determination is based 
on the settings for the repository and defaults to false.
```